xcell/cls/cl.py

```diff
@@ -226,12 +226,31 @@ def get_cl_file(self):
                 cl -= nl
             # Note that while we have subtracted the noise
             # bias from `cl_cp`, `cl_cov_cp` still includes it.
+            correction = None
```

Comment:
Do we need the correction outside of the `if` statement below? If not, I would initialize it as 1 instead (since that's what "no correction" means)

Reply:
I have removed it since we only needed to initialize it becuase we were saving it  no matter what.